mteb/tasks/PairClassification/fra/gqnli.py

```diff
@@ -0,0 +1,73 @@
+from datasets import load_dataset
+from mteb.abstasks.AbsTaskPairClassification import AbsTaskPairClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class GqnliTask(AbsTaskPairClassification):
+    metadata = TaskMetadata(
+        name="gqnli",
+        description=(
+            "Natural Language Inference on GQNLI: "
+            "predict the relation between two sentences "
+            "(0=implication, 1=neutral, 2=contradiction)."
+        ),
+        reference="https://huggingface.co/datasets/maximoss/gqnli-fr",
+        dataset={
+            "path": "maximoss/gqnli-fr",
+            "revision": "3089f0b591628692e7c4d4122804f6995380bdad"
+        },
+        type="PairClassification",
+        category="s2s",
+        modalities=["text"],
+        eval_splits=["test"],
+        eval_langs=["fra-Latn"],
+        main_score="max_accuracy",
+        date=("2025-08-05", "2025-08-05"),
```

Comment:
This date appears to be incorrect - it should be the date range during which the texts were written.